src/transformers/models/paligemma/configuration_paligemma.py

```diff
@@ -120,6 +120,12 @@ def __init__(
                 is_encoder_decoder=False,
                 vocab_size=vocab_size,
             )
+
+        # BC: `use_bidirectional_attention` was originally unset in PaliGemma1 (backbone = Gemma1) AND PaliGemma2
+        # (backbone = Gemma2). Both PaliGemmas want to default to True.
+        if self.text_config.use_bidirectional_attention is None:
+            self.text_config.use_bidirectional_attention = True
```

Comment:
needed to have a correct behavior in the situations where our mask creator simply returns `None` (i.e. when it says "use the default mask for your case", controlled by this flag)

Reply:
I don't think that's what we want TBH